docs/ResourcesContents.md

```diff
@@ -0,0 +1,51 @@
+- [route_table](https://github.com/oracle/terraform-provider-oci/tree/master/docs/resources/core/route_table.md)
+- [security_list](https://github.com/oracle/terraform-provider-oci/tree/master/docs/resources/core/security_list.md)
+- [subnet](https://github.com/oracle/terraform-provider-oci/tree/master/docs/resources/core/subnet.md)
+- [virtual_networks](https://github.com/oracle/terraform-provider-oci/tree/master/docs/resources/core/virtual_networks.md)
+- [vnic_attachment](https://github.com/oracle/terraform-provider-oci/tree/master/docs/resources/core/vnic_attachment.md)
+- [volume](https://github.com/oracle/terraform-provider-oci/tree/master/docs/resources/core/volume.md)
+- [volume_attachment](https://github.com/oracle/terraform-provider-oci/tree/master/docs/resources/core/volume_attachment.md)
+- [volume_backup](https://github.com/oracle/terraform-provider-oci/tree/master/docs/resources/core/volume_backup.md)
+
+## database
+- [db_system](https://github.com/oracle/terraform-provider-oci/tree/master/docs/resources/database/db_system.md)
+
+## identity
+- [api_key](https://github.com/oracle/terraform-provider-oci/tree/master/docs/resources/identity/api_key.md)
+- [compartment](https://github.com/oracle/terraform-provider-oci/tree/master/docs/resources/identity/compartment.md)
+- [group](https://github.com/oracle/terraform-provider-oci/tree/master/docs/resources/identity/group.md)
+- [policy](https://github.com/oracle/terraform-provider-oci/tree/master/docs/resources/identity/policy.md)
+- [swift_password](https://github.com/oracle/terraform-provider-oci/tree/master/docs/resources/identity/swift_password.md)
+- [ui_password](https://github.com/oracle/terraform-provider-oci/tree/master/docs/resources/identity/ui_password.md)
+- [user](https://github.com/oracle/terraform-provider-oci/tree/master/docs/resources/identity/user.md)
+-
+[user_group_membership](https://github.com/oracle/terraform-provider-oci/tree/master/docs/resources/identity/user_group_membership.md)
+
+
+## loadbalancer
+- [backend](https://github.com/oracle/terraform-provider-oci/tree/master/docs/resources/loadbalancer/backend.md)
+- [backendset](https://github.com/oracle/terraform-provider-oci/tree/master/docs/resources/loadbalancer/backendset.md)
+- [certificate](https://github.com/oracle/terraform-provider-oci/tree/master/docs/resources/loadbalancer/certificate.md)
+- [listener](https://github.com/oracle/terraform-provider-oci/tree/master/docs/resources/loadbalancer/listener.md)
+- [loadbalancer](https://github.com/oracle/terraform-provider-oci/tree/master/docs/resources/loadbalancer/loadbalancer.md)
+
+## objectstorage
+- [bucket](https://github.com/oracle/terraform-provider-oci/tree/master/docs/resources/objectstorage/bucket.md)
+- [object](https://github.com/oracle/terraform-provider-oci/tree/master/docs/resources/objectstorage/object.md)
+
+## About the provider
+This provider was written on behalf of Oracle by [MustWin.](http://mustwin.com/)
```

Comment:
We have this line in the README.md. Is it needed here?

Reply:
not unless you want it or there's a legal reason, I can remove it.